test/javascript.js

```diff
@@ -248,4 +249,32 @@ describe('javascript', function() {
     assert.equal(typeof output, 'function');
     assert.equal(output(), 3);
   });
+
+  it('should support requiring graphql files', async function() {
```

Comment:
can you move this to a separate test file for graphql?

Reply:
Ok, I will move